designer/client/src/components/common/DropdownButton.tsx

```diff
@@ -50,7 +54,6 @@ export function DropdownButton<T>(props: PropsWithChildren<ButtonProps & Dropdow
         >
             <Select
```

Comment:
Maybe better way to create styled component bused on that component?

Reply:
Unfortunately, not, because the dropdown menu is added to the body HTML element, so we need to set style as props here